test/tests/LayerAnimationTest.coffee

```diff
@@ -879,3 +879,73 @@ describe "LayerAnimation", ->
 				done()
 			layer.animate
 				gradient: null
+
+	describe "Border animations", (done) ->
+
+		it "should animate borders from number to number", (done) ->
+			layer = new Layer
+				borderWidth: 10
+				borderRadius: 20
+			layer.on Events.AnimationEnd, ->
+				layer.borderWidth.should.equal 30
+				layer.borderRadius.should.equal 40
```

Comment:
I understand the drive to combine these, but they should all be separated into tests for borderRadius and borderWidth